playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/pre.yml

```diff
@@ -110,8 +110,8 @@
   # are modified to use the correct image tag.  However, this can trigger a
   # docker restart if new configuration is laid down which would immediately
   # pull the latest image and defeat the purpose of these tasks.
-  - openshift_cli
-  pre_tasks:
+  - openshift_docker
```

Comment:
Yeah goal was to modify docker config, I wanted to get new registries applied during upgrade so you don't need a re-run of config.yml to do that.